### PR TITLE
Change gerrit urls for shh config support

### PR DESCRIPTION
**Pull request by jharryma**
https://ontrack-internal.amd.com/browse/ROCMOPS-7457

Needs to be merged before: https://github.com/ROCm/rocm-spack/pull/170

Allows github runners to use private gerrit repo

### CODE REVIEW

**Review by renjithravindrankannath — APPROVED** [review]
LGTM

File: var/spack/repos/builtin/packages/rocm-device-libs/package.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class RocmDeviceLibs(CMakePackage):
     """set of AMD specific device-side language runtime libraries"""
 
     homepage = "https://github.com/ROCm/ROCm-Device-Libs"
-    git = "ssh://<user-id>@gerrit-git.amd.com:<port-number>/lightning/ec/device-libs.git"
+    git = "ssh://gerritgit/lightning/ec/device-libs.git"
```
Comment (srekolam) [inline var/spack/repos/builtin/packages/rocm-device-libs/package.py:14]: rocm-device-libs is now part of the llvm-project. perhaps this and comgr needs to be pointing to gerritgit/lightning/ec/llvm-project.git.

**Review by srekolam — APPROVED** [review]
overall replacing user-id@gerrit-git.amd.com:port-number with gerritgit is okay i think.

File: var/spack/repos/builtin/packages/rocm-device-libs/package.py (hunk 1 above)
Comment (jharryma) [reply var/spack/repos/builtin/packages/rocm-device-libs/package.py:14]: Can this one be address in a different PR?

Comment (renjithravindrankannath) [reply var/spack/repos/builtin/packages/rocm-device-libs/package.py:14]: Yes. We need to update the branch name for llvm-amdgpu as well. We can update the URL as well with that changes.

**Pull request merged**